### PR TITLE
mgr/dashboard: Extend "Writing End-to-End Tests" section (describe vs it)

### PR DESCRIPTION
**Pull request by rafaelquint**
Fixes: https://tracker.ceph.com/issues/40395

Signed-off-by: Adam King <adking@redhat.com>
Signed-off-by: Rafael Quintero <rquinter@redhat.com>

### CODE REVIEW

**Comment by tspmelo** [comment]
jenkins test dashboard

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 1]
```diff
@@ -261,6 +261,59 @@ Please refer to the official `Protractor style-guide
...
+    images.editImage(imageName, poolName, newImageName, '2');
+    expect(images.getTableCell(newImageName).isPresent()).toBe(true);
+  });
+  //...
+ });
+
+As shown from the previous example, our ``describe()`` test suite is to create, edit
+and delete an image. Therefore, each ``it()`` completes one of these steps, one for creating,
+one for editing, and so on. Likewise, every ``it()`` blocks message should be in lowercase
+and begin with the world "should". It should describe what the individual test is doing and
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/HACKING.rst]: `should` doesn't have to be a hard requirement, as long as `it()` can be the prefix of the message `it()'s message is fine ;)

For example:
`it('edits the test image`, () => ...)`

Comment (Devp00l) [reply src/pybind/mgr/dashboard/HACKING.rst]: You could replace "it" with the function call. This way you present the reader how `it()` can be used :)
```suggestion
and begin with the world "should". `it()` should describe what the individual test is doing and
```

Comment (rafaelquint) [reply src/pybind/mgr/dashboard/HACKING.rst]: Thank you for the recommendation, change should be made in most recent push.

**Comment by rafaelquint** [comment]
jenkins test make check

**Comment by rafaelquint** [comment]
jenkins test dashboard

**Comment by alfonsomthd** [comment]
jenkins test docs

**Comment by adk3798** [comment]
Extended the title underline on line 265 since it was causing a linting error

**Comment by adk3798** [comment]
jenkins test submodules

**Comment by adk3798** [comment]
jenkins retest this please

**Comment by adk3798** [comment]
jenkins test submodules

**Comment by adk3798** [comment]
Is there a Jenkins command to rerun the Signed-of-by check? It failed due to being unable to fetch a remote repo and I believe the sign-off on this PR is done correctly.

**Comment by tspmelo** [comment]
I think you need to push again.

BTW: You have a typo on the commit message, the 1st `"` is in the wrong position.
Replace `mgr/dashboard: "Extend Writing End-to-End Tests" section (describ...` with `mgr/dashboard: Extend "Writing End-to-End Tests" section (describ...`

**Comment by adk3798** [comment]
Fixed the commit message typo @tspmelo  mentioned

**Comment by adk3798** [comment]
jenkins test make check

**Pull request merged**